packages/test-utils/src/index.ts

```diff
@@ -8,7 +8,7 @@ import supertest from 'supertest-light';
 // @ts-ignore
 import { Keystone } from '@keystone-next/keystone-legacy';
 import { initConfig, createSystem, createExpressServer } from '@keystone-next/keystone';
-import { runPrototypeMigrations } from '@keystone-next/adapter-prisma-legacy';
+import { pushPrismaSchemaToDatabase } from '@keystone-next/keystone/migrations';
```

Comment:
Note no changeset for this because there hasn't yet been a release where the test utils used this util from the adapter package.